bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -0,0 +1,180 @@
+        }
+        String cameraId = homeEvent.getCameraId();
+        handler.getActiveChildren().stream().filter(handler -> cameraId.equals(handler.getId())).findFirst()
+                .ifPresent(handler -> handler.setNewData(homeEvent));
+    }
+
+    Collection<HomeEvent> getCameraEvents(String cameraId) {
+        return api.map(api -> {
+            try {
+                return api.getCameraEvents(handler.getId(), cameraId);
+            } catch (NetatmoException | NoSuchElementException e) {
+                logger.warn("Error retrieving last events of camera '{}' : {}", cameraId, e.getMessage());
+                return null;
+            }
+        }).orElse(List.of());
+    }
+
+    Collection<HomeEvent> getPersonEvents(String personId) {
+        return api.map(api -> {
+            try {
+                return api.getPersonEvents(handler.getId(), personId);
+            } catch (NetatmoException | NoSuchElementException e) {
+                logger.warn("Error retrieving last events of person '{}' : {}", personId, e.getMessage());
+                return null;
+            }
+        }).orElse(List.of());
+    }
+
+    void setPersonAway(String personId, boolean away) {
+        api.ifPresent(api -> {
+            try {
+                api.setPersonAwayStatus(handler.getId(), personId, away);
+                handler.expireData();
+            } catch (NetatmoException | NoSuchElementException e) {
+                logger.warn("Error setting person away/at home '{}' : {}", personId, e.getMessage());
+            }
+        });
+    }
+
+    @Nullable
+    String ping(String vpnUrl) {
+        return api.map(api -> {
+            try {
+                return api.ping(vpnUrl);
+            } catch (NetatmoException | NoSuchElementException e) {
+                logger.warn("Error pinging camera '{}' : {}", vpnUrl, e.getMessage());
+                return null;
+            }
+        }).orElse(null);
+    }
+
+    void changeStatus(@Nullable String localURL, boolean status) {
+        api.ifPresent(api -> {
+            if (localURL != null) {
+                try {
+                    api.changeStatus(localURL, status);
+                    handler.expireData();
+                } catch (NetatmoException e) {
+                    logger.warn("Error changing camera monitoring status '{}' : {}", localURL, e.getMessage());
+                }
+            } else {
+                logger.info("Monitoring can only be done on local camera.");
+            }
+        });
+    }
+
+    void changeFloodlightMode(@Nullable String localURL, FloodLightMode mode) {
+        api.ifPresent(api -> {
+            if (localURL != null) {
+                try {
+                    api.changeFloodLightMode(localURL, mode);
+                    handler.expireData();
+                } catch (NetatmoException e) {
+                    logger.warn("Error changing Presence floodlight mode '{}' : {}", localURL, e.getMessage());
```

Comment:
Please confirm that you want to display localURL and not mode as first parameter

Reply:
Right also.